api/coreservice.go

```diff
@@ -945,6 +948,7 @@ func (core *coreService) readState(ctx context.Context, p protocol.Protocol, hei
 			// old data, wrap to history state reader
 			d, h, err := p.ReadState(ctx, factory.NewHistoryStateReader(core.sf, rp.GetEpochHeight(inputEpochNum)), methodName, arguments...)
 			if err == nil {
+				key.Height = strconv.FormatUint(h, 10)
```

Comment:
above places use `strconv.ParseUint(height, 0, 64)`, which is different from base10, can you check and verify?